Makefile

```diff
@@ -5,38 +5,34 @@ CCFLAGS = -Wall -c
 
 ifeq ($(OS),Windows_NT)
 	os = windows
-	CCFLAGS += -D WIN32
+	CCFLAGS += -D WIN32 -D IA32
```

Comment:
We only distribute 32 bit windows executables of flashlight, so there's no point in building 64 bit.